apps/src/studio/studio.js

```diff
@@ -261,6 +261,23 @@ function loadLevel() {
   Studio.MAZE_HEIGHT = Studio.SQUARE_SIZE * Studio.ROWS;
   studioApp.MAZE_WIDTH = Studio.MAZE_WIDTH;
   studioApp.MAZE_HEIGHT = Studio.MAZE_HEIGHT;
+
+  if (skin.wallMapLayers) {
+    Studio.wallMapLayers = {};
+    for (let map in skin.wallMapLayers) {
+      let img = new Image();
+      img.onload = function () {
+        var canvas = document.createElement('canvas');
+        canvas.width = img.width;
+        canvas.height = img.height;
+        var context = canvas.getContext('2d');
+        context.drawImage(img, 0, 0);
+        Studio.wallMapLayers[map] =
+            context.getImageData(0, 0, canvas.width, canvas.height).data;
+      };
+      img.src = skin.wallMapLayers[map].srcUrl;
```

Comment:
It would be nice to extract this whole `imageDataFromSourceUrl` process into a helper function.  I've got a few of these in gamelab/animationListModule.js, but I don't think any of them do exactly what you want.  Maybe we can combine those helpers into a file of image manipulation utilities?